config/locales/en.yml

```diff
@@ -935,7 +937,9 @@ en:
         trendable: Can appear under trends
         trending_rank: 'Trending #%{rank}'
         usable: Can be used
-        usage_comparison: Used %{today} times today, compared to %{yesterday} yesterday
+        usage_comparison:
+          one: Used %{today} time today, compared to %{yesterday} yesterday
+          other: Used %{today} times today, compared to %{yesterday} yesterday
```

Comment:
This would not actually work, rails-i18n pluralization only works with a `count` key afaik. So, `today` would have to be replaced with `count`, and the call sites changed accordingly. I am not sure this is worth it.